website/source/docs/guides/forwarding.html.md

```diff
@@ -155,6 +156,32 @@ You may have to add the following line to the bottom of your
 include: "/etc/unbound/unbound.conf.d/*.conf"
 ```
 
+### systemd-resolved Setup
+
+`systemd-resolved` is typically configured with `/etc/systemd/resolved.conf`. 
+To configure systemd-resolved to send queries for the consul domain to
+Consul, configure resolved.conf to contain the following:
+
+```
+DNS=127.0.0.1
+Domains=~consul
+```
+
+The main limitation with this configuration is that the DNS field
+cannot contain ports. So for this to work either Consul must be
+[configured to listen on port 53](https://www.consul.io/docs/agent/options.html#dns_port)
+instead of 8600 or you can use iptables to map port 53 to 8600. 
+The following iptables commands are sufficient to do the port
+mapping.
+
+```
+[root@localhost ~]# iptables -t nat -A OUTPUT -d localhost -p udp -m udp --dport 53 -j REDIRECT --to-ports 8600
+[root@localhost ~]# iptables -t nat -A OUTPUT -d localhost -p tcp -m tcp --dport 53 -j REDIRECT --to-ports 8600
+```
```

Comment:
How about using the expanded flags, making it easier to understand.

```
iptables --table nat --append OUTPUT --protocol udp --destination localhost --match udp --dport 53 --jump REDIRECT --to-ports 8600
```

1. Also, I'm curious if the `--match` flag is needed?
2. Would there be any benefit to adding `--out-interface lo`, to restrict this to the loopback interface?

Reply:
So all I did was copy two of the necessary iptables rules from the iptables section of the forwarding guide which use the shortened -t -A -d etc.

As for the --match and --out-interface: --out-interface should be redundant as the destination being matches is localhost. Unless you have configured localhost to map to another IP or are doing something very wrong with your routing this should be adequate to restrict it to the loopback interface. The -m flags are definitely needed in order to only forward 53 to 8600 and not all UDP traffic.

Reply:
Sounds good!